### PR TITLE
Add flex recipe for SuluRedirectBundle

### PR DESCRIPTION
**Pull request by luca-rath**
| Q             | A
| ------------- | ---
| License       | MIT

Link to package: https://github.com/sulu/SuluRedirectBundle

### CODE REVIEW

**Comment by luca-rath** [comment]
The CI is failing because there is no `phpcr/phpcr-implementation` available. This is required in sulu's skeleton, see https://github.com/sulu/skeleton/blob/2.x/composer.json#L38

See https://github.com/symfony/recipes-contrib/pull/995#issuecomment-682479149

**Review by alexander-schranz — APPROVED** [review]

File: sulu/redirect-bundle/2.0/post-install.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<bg=blue;fg=white>              </>
+<bg=blue;fg=white> What's next? </>
+<bg=blue;fg=white>              </>
+
+  * The <fg=green>SuluRedirectBundle</> is almost ready:
+    1. Print the sql statements required to update your database schema by running <info>php bin/console doctrine:schema:update --dump-sql</>.
+    2. If the statements look fine to you, run the same command using <info>--force</> to execute them.
+    3. Make sure your user has all the necessary permissions for the <fg=green>SuluRedirectBundle</>.
+
+  * <fg=blue>Read</> the documentation at <comment>https://github.com/sulu/SuluRedirectBundle/blob/2.x/Resources/doc/installation.md</>
```
Comment (luca-rath) [inline sulu/redirect-bundle/2.0/post-install.txt:10]:
```suggestion
  * <fg=blue>Read</> the documentation at <comment>https://github.com/sulu/SuluRedirectBundle/blob/2.x/Resources/doc/README.md</>
```

**Comment by alexander-schranz** [comment]
@Nyholm would be great if you could have a look at this and https://github.com/symfony/recipes-contrib/pull/1098

**Comment by Nyholm** [comment]
I think it looks good. Could you help me understand why the CI is so red?

**Comment by luca-rath** [comment]
> I think it looks good. Could you help me understand why the CI is so red?

It's explained here https://github.com/symfony/recipes-contrib/pull/1114#issuecomment-804997276

**Review by Nyholm — APPROVED** [review]
Oh, I see. 

Thank you

**Comment by luca-rath** [comment]
thank you! 🙂

**Pull request merged**